testplan/web_ui/testing/src/Nav/NavList.js

```diff
@@ -1,67 +1,115 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import _ from 'lodash';
+import base64url from 'base64url';
 
+import InteractiveNavEntry from './InteractiveNavEntry';
 import NavEntry from './NavEntry';
 import {CreateNavButtons, GetNavColumn} from './navUtils.js';
-import {STATUS} from "../Common/defaults";
+import {
+    STATUS,
+    RUNTIME_STATUS,
+    NAV_ENTRY_ACTIONS,
+} from "../Common/defaults";
 
 /**
  * Render a vertical list of all the currently selected entries children.
+  * Render a vertical list of all the currently selected entries children for
```

Comment:
Looks like sort of duplicate with the line above.

Reply:
Indeed. Fixing, thanks @zhenyu-ms for catching it.